### ISSUE TITLE
React js cdn

### ISSUE THREAD

**Issue by Wilmerj** [paymentez/paymentez.js#11]
Hi, They know how to use cdn in one component of react js?

I use `window.Paymentez.init('stg', '', ''); `

but dont work , it tells me an error

> Cannot read property 'init' of undefined

**Comment by luanlucho** [comment]
Same problem here!!!

**Comment by luanlucho** [comment]
@Wilmerj were you able to find a fix?

I called `window.Paymentez.init('stg', '', '');` from `componentDidMount()`...  I don't get any errors here, but when I call:

```
const myCard = $('#my-card');
const cardToSave = myCard.PaymentezForm('card');
```
I get `myCard.PaymentezForm is not a function`

Please Help!!!

**Comment by pidmindiz** [comment]
@Wilmerj @luanlucho Has anyone been successful in the implementation?

**Comment by donrich2020** [comment]
It doesn't work with Angular neither.

**Comment by hextiandro** [comment]
Hi, I have the same problem.. Has anyone been successful in the implementation?. Please.

**Comment by manunoly** [comment]
i had to implement this by my own, never find one working example with IONIC or Angular, any luck with react??

**Comment by djra26111990** [comment]
> i had to implement this by my own, never find one working example with IONIC or Angular, any luck with react??

how do you resolve this? you have one example to share with us @manunoly ??

**Comment by manunoly** [comment]
Yes i have a working solution with IONIC, i will share how i do it in a demo project because is a private client.
